### DISCUSSION TITLE
About the process of releasing new version.

### DISCUSSION THREAD

**Discussion by AnguseZhang** [deepmodeling/dpdispatcher#294] [General]
Dear developers, 

  We notice that `dpdispatcher` has been a fundamental infrastructure. We hope that each time before `dpdispatcher` releases a new version, the developers of Bohrium can perform some necessary tests and make sure that the tests pass. @AcoderHIT and  @KZHIWEI will support this. Also, they will help add unit tests for relevant interface. 
 
   What do you think? @njzjz @felix5572.

**Comment by njzjz** [comment]
Could Bohrium provide a server image for tests?

**Reply by AnguseZhang** [reply]
Definitely. @ACoderHIT Please have a look.
